MediaGalleryUi/Model/FilterParametersProviderComposite.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Model;
+
+use Magento\Framework\DB\Select;
+use Magento\Framework\Api\SearchCriteriaInterface;
+
+/**
+ * A class that keeping the list of asset list filters
+ */
+class FilterParametersProviderComposite implements SelectModifierInterface
```

Comment:
Let's rename all occurrences of `FilterParametersProvider` to `SelectModifier`
```suggestion
class SelectModifierComposite implements SelectModifierInterface
```